bin/templates/cordova/lib/run.js

```diff
@@ -41,8 +41,9 @@ var path  = require('path'),
 
     var self = this;
 
-    var install_target = runOptions.device ? '--device' :
-        runOptions.emulator ? '--emulator' :
+    var install_target = !runOptions.target ?
```

Comment:
IMO this statement looks very complex - there is 3 ternary operators in one expression. Could you please try to simplify this